### DISCUSSION TITLE
else clause in switch

### DISCUSSION THREAD

**Discussion by Inc0n** [odin-lang/Odin#1996] [Ideas/Requests]
It would be slightly more convienient to have a else clause in switch statements, which would unconditionally fell through:

For example: 

```
switch value {
  case 0: do something 
  case 1: do some other thing
  else:
}
```
However, it can be done in a different way, right now:
```
switch {
  case value == 0: do something 
  case value == 1: do some other thing
  case true: do something about value here
}
```

**Comment by Kelimion** [comment]
We already have that:
```cpp
switch value {
  case 0: do something 
  case 1: do some other thing
  case:  do something for all other cases
}
```

**Reply by Inc0n** [reply]
Thank you and my bad, it did not work once for some other reasons and I thought it was not doable.
